api/src/main/java/com/codeforcommunity/dto/site/UpdateSiteRequest.java

```diff
@@ -0,0 +1,440 @@
+package com.codeforcommunity.dto.site;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UpdateSiteRequest extends ApiDto {
+  private Boolean tree_present;
```

Comment:
It looks like we're using camel case in other DTOs. @florisdobber does it matter for this?

Reply:
Yes, this should be camelCase